File: abc/abc161/abc161_a.cs
// https://atcoder.jp/contests/abc161/submissions/11546573

using System;
using System.Linq;

namespace CSharp
{
    class Program
    {
        static void Main()
        {
            var input = Console.ReadLine().Split().Select(long.Parse).ToArray();
            var X = input[0];
            var Y = input[1];
            var Z = input[2];

            Console.WriteLine($"{Z} {X} {Y}");
        }
    }
}

File: abc/abc161/abc161_a.fs
// https://atcoder.jp/contests/abc161/submissions/11738710

let input = stdin.ReadLine().Split()
let X = input.[0]
let Y = input.[1]
let Z = input.[2]

[ Z; X; Y ]
|> String.concat (" ")
|> printfn "%s"
